docs/data-model.md

```diff
@@ -0,0 +1,29 @@
+# alerts_concierge Data Model
+
+## User-Driven
+
+In alerts_concierge, *trips* (referred to in the UI as “subscriptions”) represent an entire commute, potentially including transfers and a return journey.
+
+A trip is composed of multiple *subscriptions*, representing an individual leg of a commute; subscriptions have a single time range, a single route, and a single pair of stops where they start and end.
+Transfers create different subscriptions on the same trip (with a higher `rank`), the Green Line trunk creates different subscriptions on the same trip (one per `route`), and return trips create different subscriptions on the same trip (with `return_trip` `true` and the opposite `direction_id`).
+
+For commuter rail and ferry, to support riders who plan around individual scheduled trips, the UI presents a list of trips during the selected time range, appearing to allow the individual scheduled trips themselves to be included or excluded from the subscription.
+However, the subscription only tracks this information in `travel_start_time`/`travel_end_time`, which is set in the interface to between the first and the last selected trip, and indeed the interface does not allow the selection of non-contiguous sets of trips.
+Storing selected trips by ID would lead to selections losing meaning at new ratings if trip IDs change; this approach allows an equivalent trip at the same time to remain selected if its ID changes in the new rating.
+
+A trip can instead represent a non-time-windowed interest in accessibility information (`trip_type` of `accessibility` rather than `commute`, and subscription `type`s of `accessibility` rather than `subway`/`bus`/`cr`/`ferry`).
```

Comment:
question: Can a "travel" trip also include accessibility subscriptions? I know we present it that way in the UI, but can't remember if we separate out multiple data structures on the back end.

Reply:
Yes, it can. Rephrased this paragraph.